.github/workflows/artifact.yml

```
@@ -113,20 +111,22 @@ jobs:
 
       - name: Upload to GitHub Release
         if: ${{ github.event.inputs.upload_release }}
+        permissions:
```

actionlint (1.7.12)

[error] unexpected key "permissions" for step to run shell command. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "working-directory"

(syntax-check)